documentation/working-with-testcafe-studio/reports.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: docs
+title: Reports
+permalink: /documentation/working-with-testcafe-studio/reports
+---
+# Reports
+
+You can view results for all executed test runs in the **Reports** panel. Each row in the panel corresponds to an individual test run (it can be a single test, a fixture's test run or an entire directory's test run) and contains the following information: the test run's progress, start time, duration and the number of tests that have failed and passed. Different colors indicates the test run status: green - passed tests, red - failed tests, blue - running and pending tests.
```

Comment:
`it can be a single test, a fixture's test run or an entire directory's test run`
It can be group of items(test, fixtures, dirs) as well. Probably we need to rephrase this sentence.